source/javascript/src/navBar.js

```diff
@@ -0,0 +1,44 @@
+import { DateConverter } from './utils/DateConverter.js'
+import { Router, ROUTES } from './utils/Router.js'
+import { IndexedDBWrapper } from './indexedDB/IndexedDBWrapper.js'
+
+document.addEventListener('DOMContentLoaded', (event) => {
+  const indexNavButton = document.querySelector('.nav-index')
+  const searchNavButton = document.querySelector('.nav-search')
+  const dailyNavButton = document.querySelector('.nav-daily')
+  const collectionNavButton = document.querySelector('.nav-collection')
+  const weeklyNavButton = document.querySelector('.nav-weekly')
+
+  const defaultNav = (route, params = new URLSearchParams()) => {
+    const url = new URL(route, location.origin)
```

Comment:
location.origin might be a problem for github pages.
For example, the location.origin is `https://cse110-w21-group7.github.io/` and if we add anything after that, it would throw a 404

Reply:
For the sake of demo video I'm going to merge it, but we do need to make a note of it and come back to it

Reply:
GitHub Pages doesn't allow for query parameters then?